dsperse/src/compile/utils/compiler_utils.py

```
@@ -0,0 +1,273 @@
+                    if not os.path.exists(slice_path):
+                        logger.warning(f"Slice file not found for index {idx}: {slice_path}")
+                        continue
+                else:
+                    logger.error(f"No valid path found for slice index {idx}")
+                    continue
+
+                slice_output_path = os.path.join(os.path.dirname(slice_path), "ezkl")
+                os.makedirs(slice_output_path, exist_ok=True)
+
+                # Run ONNX inference to generate calibration data
+                output_tensor_path = os.path.join(slice_output_path, f"calibration.json")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)